src/fusor/models.py

```diff
@@ -63,12 +60,10 @@ class FunctionalDomain(BaseModel):
 
     type: Literal[FUSORTypes.FUNCTIONAL_DOMAIN] = FUSORTypes.FUNCTIONAL_DOMAIN
     status: DomainStatus
-    associated_gene: GeneDescriptor
+    gene: Gene
```

Comment:
Is there a reason why we're renaming `associated_gene` to `gene`? I didn't look at the spec, so not sure if it was changed there

Reply:
I changed it based on Jeremy's update to the example here: https://github.com/cancervariants/fusor/pull/151/files#diff-f4604af4ef1e1fcc60f0c63b08f06427ff72f5a2541afc048c965e34aae6ab76L83-R115

Reply:
@jarbesfeld Is this part of the fusion spec?

Reply:
This should be `associated_gene` not `gene`(https://fusions.cancervariants.org/en/latest/information_model.html#functional-domains)